packages/core/src/components/Spinner/Spinner.scss

```diff
@@ -113,11 +113,22 @@ Style guide: components.spinner
 
 - If the process takes a long time, use something else.
 
+## Accessibility Considerations
+
+- Spinners are useful to announce a page load sequence, and should be paired with a parent container that includes several ARIA attributes:
```

Comment:
Should we add these ARIA attributes to the examples that show spinners within a parent container? (Button, "Filled", and the React component examples)

Reply:
I think we should pass on adding these attributes to the buttons. It's more useful in context of a containing element like a div or table, that when updated, announces to users. I don't believe a button, even with the extra aria markup, will announce a change in text.

If you'd like to add the extra markup to the buttons, I'll do some further testing on that scenario.